### DISCUSSION TITLE
AlDente 1.07

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#355] [General]
- Discharge in Clamshell Mode is now possible for Pro users!
- Fixed a bug regarding resetting the SMC values when uninstalling the helper.
- Fixed a bug that caused textboxes to not show numbers.
- Topup and Calibration Mode now persist when your MacBook goes to sleep.
- Many other small fixes and changes.

[Hope you like our Pasta,](https://www.youtube.com/watch?v=Wn4Kxy6bPI4)
David & Matthias

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente/releases/tag/1.07'>AlDente 1.07</a>.</em>

**Comment by tomwolsky** [comment]
Not having luck getting AlDente to work in Monterey. I have the Charge Limit set to 50%, but the battery level has creeped up past it and says 1h 44m to fully charged.

Thanks

<img width="251" alt="60%_Screen Shot 2021-08-23 at 14 23 19" src="https://user-images.githubusercontent.com/18217685/130507841-a6126aa7-df1e-485a-b4f8-e3626579d2c4.png">
<img width="168" alt="60%_Screen Shot 2021-08-23 at 14 23 26" src="https://user-images.githubusercontent.com/18217685/130507846-debd163f-4b9b-4d14-aae1-34255ad6e322.png">

**Comment by owenjuni** [comment]


**Reply by MatthiasKerbl** [reply]
> 1.01版Pro疯狂地报错（一直需要安装帮助程序），免费版为1.07，而Pro版还在1.01版，一升级后就变成免费版
> https://owenjuni-cos-1256992750.cos.ap-guangzhou.myqcloud.com/uPic/ohL94M.mov

Hi @owenjuni,

Please uninstall all older versions of AlDente and reinstall the newest version AlDente 1.07.

Cheers,
Matthias

**Comment by Archer785** [comment]
I cant seem to find the discharge button and it is not working for me automatically, I have MB Pro 13 inch, 2019, OS X 11.6 and the newest 1.07 installed.
Am I doing something wrong?

**Reply by MatthiasKerbl** [reply]
Hi @Archer785,

The Discharge button is above the percentage slider.

Cheers,
Matthias

**Reply by Archer785** [reply]
@MatthiasKerbl  thank you for the answer - I have discovered that discharge button is only available when not using the Intel mode, hence my mistake :)
